ag/validation.go

```diff
@@ -49,6 +52,23 @@ func Interceptor(logger *zap.Logger) grpc.UnaryServerInterceptor {
 		}
 		ctx, cancel := context.WithTimeout(ctx, MaxWait)
 		defer cancel()
+
+		meta, ok := metadata.FromIncomingContext(ctx)
```

Comment:
Ideally, this logic should be in a separate interceptor. See comment in main.go.